src/MainView.mc

```diff
@@ -98,11 +104,27 @@ class MainView extends WatchUi.View {
     }
   }
 
+  function drawProgress(dc, value, max, codeColor) {
+    dc.setPenWidth(dc.getHeight() / 40);
+    dc.setColor(codeColor, Graphics.COLOR_TRANSPARENT);
+    if (screen_shape_== System.SCREEN_SHAPE_ROUND) {
+      if ( _enableAntiAlias ) {
```

Comment:
Instead of the global variable, we should a local one and can determine each time whether anti-aliasing is possibe or not as well.. i.e. change the `if (_enableAntiAlias)` to directly `if (dc has :setAntiAlias)`.